_data/changelogs/component-input-prefix-suffix.yml

```diff
@@ -2,6 +2,12 @@ title: Input prefix or suffix
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Replaced deprecated `xlink:href` references with `href`.
+    affectsMarkup: true
+    githubPr: 6165
+    githubRepo: uswds
+    versionUswds: 3.9.0
```

Comment:
```suggestion
    versionUswds: N.N.N
```

Reply:
Resolved in [3ec6a6d](https://github.com/uswds/uswds-site/pull/2951/commits/3ec6a6de4ee0a7bf58b8a216a58233d2f7869d7e)